versions/latest/modules/en/pages/rancher-admin/scc-registration.adoc

```diff
@@ -0,0 +1,107 @@
+= SUSE Customer Center Registration
+:revdate: 2025-08-19
+:page-revdate: {revdate}
+
+Since Rancher v2.12.1, you should now register your SUSE Rancher Manager Prime Suite with https://scc.suse.com/home[SUSE Customer Center (SCC)] from the Rancher UI or through Helm. This will enable you to see your registered systems in SCC under your subscription. 
+
+== Prequsities
+
+* Registering your Rancher Manager instance with SCC requires a SUSE account. If you don't have a SUSE account yet, visit the https://scc.suse.com/[SCC home page] to create one.
+
+* You must have access to your organization. Ask your organization's administrator to add you to the account. See the https://scc.suse.com/docs/userguide#UG-Requesting-Access-to-an-Organizations-Account[SCC documentation] for details. If you know your organization's registration code, enter it when prompted.
+
+* You must have the registration code for your Rancher Manager subscription on SCC. To get the registration code:
+. Log in to SCC.
+. Navigate to **My Organizations** and click on the organization with your Rancher Manager subscription.
+. Find your Registration code under *Organization* > *Subscriptions* > *Subscription Information*.
+
+== Register with SUSE Customer Center
+
+From the Rancher UI, check the Notification Center (bell icon) in the upper-right corner for a "Rancher Prime Registration" notification and click *Register Now*. This redirects you to the *Registration* page under *Global Settings*. You can also directly go to *Global Settings* > *Registration*.
```

Comment:
We should list the actual navigation path as the default/first choice. The last sentence says it's the direct path, but we're listing the indirect option first.

Also, if the "Rancher Prime Registration" notification behaves like other notifications (is it a special notification?) then it can disappear after someone marks it as read.